distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -853,7 +853,7 @@ Text.take : (Text_Sub_Range | Index_Sub_Range | Range | Integer) -> Text ! Index
 Text.take self range=(First 1) =
     ranges = Text_Sub_Range_Module.find_codepoint_ranges self range
     case ranges of
-        Range_Data start end _ ->
+        Range.Value start end _ ->
```

Comment:
```suggestion
        Range.Value start end 1 ->
```

It seems that now the following will work :grin: not sure if it's thanks to the new parser or some work on pattern matching, but it works as expected, see for example the following snippet:

```
from Standard.Base import all

foo r = case r of
    Range_Data a b 1 -> IO.println a.to_text+"-->"+b.to_text
    _ -> IO.println "other: "+r.to_text

main =
    foo (0.up_to 10)
    foo (0.up_to 15 . with_step 2)
```

it runs fine on latest develop and prints
```
0-->10
other: (Range_Data 0 15 2)
```

I know it's not directly part of this task, but given that this code is being touched anyway. I think it would be a good way of asserting that the step is indeed 1 (it is an assumption provided by the `find_codepoint_ranges` method.